SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -979,6 +996,34 @@ void ActorSystem::ApplyComponentData(USpatialActorChannel& Channel, UObject& Tar
 	}
 }
 
+void ActorSystem::ResolvePendingOpsFromEntityUpdate(TArray<FWeakObjectPtr> ToResolveOps)
```

Comment:
const ref

Reply:
if this was javascript this would be fine so ha

Reply:
(changed)